service/create/service.go

```diff
@@ -1182,6 +1182,32 @@ func (s *Service) deleteFunc(obj interface{}) {
 		s.logger.Log("info", "deleted public subnet")
 	}
 
+	// Before the security groups can be deleted any rules referencing other
```

Comment:
Here is where any rules referencing other groups are deleted before the groups are deleted.